onnxruntime/core/providers/dml/DmlExecutionProvider/src/External/DirectMLHelpers/DmlSerializedGraphDesc.h

```diff
@@ -0,0 +1,73 @@
+//-----------------------------------------------------------------------------
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//
+//-----------------------------------------------------------------------------
+
+#pragma once
+
+struct ConstantName
+{
+    std::string name;
+};
+
+struct ConstantData
+{
+    byte* data;
```

Comment:
>byte [](http://example.com/codeflow?start=4&length=4)

`std::byte`, consistent with `std::string` and `std::vector`. Somebody must have added a `using namespace std` somewhere along the include chain ☹.